web/src/components/apps/AppVersionHistory.jsx

```diff
@@ -637,7 +631,9 @@ class AppVersionHistory extends Component {
     // This is kinda hacky. This finds the equivalent downstream version because the midstream
     // version type does not contain metadata like version label or release notes.
     const currentMidstreamVersion = versionHistory.find(version => version.parentSequence === app.currentVersion.sequence) || app.currentVersion;
-
+    const otherAvailableVersions = versionHistory.filter((i, idx) => idx !== 0);
```

Comment:
could do this instead
```suggestion
    const otherAvailableVersions = versionHistory..slice(1);
```